.storybook/manager-head.html

```diff
@@ -148,6 +148,11 @@
       }
     }
 
+    const canvasButton = document.getElementsByClassName("css-14o7vtb");
```

Comment:
I don't understand this change. 
When I test this query against the deployed version of mgt.dev for this branch I get no matching elements.

Reply:
@gavinbarron It's related to the storybook accessibility issue [here](https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/1642) where focus goes into the canvas button, which is not interactive. Trying a fix to remove the focus from the storybook class itself, though this fix is not currently working. The css fix was to remove the outline on focus-visible but the element still gets tabbed into.

Reply:
Thanks @Mnickii looks like the class on the Canvas button changed to `sto-14o7vtb` 
This probably happened when the Storybook version changed and the color contrast on the html syntax coloring regression was introduced.

Reply:
Removed this change to investigate further, will include a fix in a separate PR